src/frontend/components/LiveVideoPanel/index.spec.tsx

```diff
@@ -0,0 +1,110 @@
+
+import { LiveVideoPanel } from '.';
+
+const mockVideo = videoMockFactory({
+  xmpp: {
+    bosh_url: 'https://xmpp-server.com/http-bind',
+    websocket_url: null,
+    conference_url:
+      '870c467b-d66e-4949-8ee5-fcf460c72e88@conference.xmpp-server.com',
+    prebind_url: 'https://xmpp-server.com/http-pre-bind',
+    jid: 'xmpp-server.com',
+  },
+});
+jest.mock('data/appData', () => ({
+  appData: {
+    video: mockVideo,
+  },
+}));
+
+describe('<LiveVideoPanel />', () => {
+  it('closes the panel if no item is selected', () => {
+    const mockSetPanelVisibility = jest.fn();
+    useLivePanelState.setState({
+      currentItem: undefined,
+      availableItems: [
+        LivePanelItem.APPLICATION,
+        LivePanelItem.CHAT,
+        LivePanelItem.JOIN_DISCUSSION,
+      ],
+      setPanelVisibility: mockSetPanelVisibility,
+    });
+
+    const { container } = render(
+      wrapInIntlProvider(<LiveVideoPanel video={mockVideo} />),
+    );
+
+    expect(mockSetPanelVisibility).toBeCalled();
+    expect(mockSetPanelVisibility).toBeCalledTimes(1);
+    expect(mockSetPanelVisibility).toBeCalledWith(false);
+
+    expect(container.hasChildNodes()).toBe(false);
+  });
+
+  it('render the content with selection', () => {
+    useLivePanelState.setState({
+      currentItem: LivePanelItem.APPLICATION,
+      availableItems: [
+        LivePanelItem.APPLICATION,
+        LivePanelItem.CHAT,
+        LivePanelItem.JOIN_DISCUSSION,
+      ],
+    });
+
+    render(wrapInIntlProvider(<LiveVideoPanel video={mockVideo} />));
+
+    screen.getByRole('tablist');
+    screen.getByRole('tab', { name: 'application' });
+    screen.getByRole('tab', { name: 'chat' });
+    screen.getByRole('tab', { name: 'viewers' });
+
+    screen.getByText('application content');
+  });
+
+  it('does not render tabs with only one item available', () => {
+    useLivePanelState.setState({
+      currentItem: LivePanelItem.APPLICATION,
+      availableItems: [LivePanelItem.APPLICATION],
+    });
+
+    render(wrapInIntlProvider(<LiveVideoPanel video={mockVideo} />));
+
+    expect(screen.queryByRole('tablist')).toBeNull();
+    expect(screen.queryByRole('tab', { name: 'application' })).toBeNull();
+    expect(screen.queryByRole('tab', { name: 'chat' })).toBeNull();
+    expect(screen.queryByRole('tab', { name: 'viewers' })).toBeNull();
+
+    screen.getByText('application content');
+  });
+
+  it('does not render tabs on small screens', () => {
+    useLivePanelState.setState({
+      currentItem: LivePanelItem.APPLICATION,
+      availableItems: [
+        LivePanelItem.APPLICATION,
+        LivePanelItem.CHAT,
+        LivePanelItem.JOIN_DISCUSSION,
+      ],
+    });
+
+    const tree = renderer
+      .create(wrapInIntlProvider(<LiveVideoPanel video={mockVideo} />))
+      .toJSON();
+
+    expect(
+      (tree as renderer.ReactTestRendererJSON).children![0],
+    ).toHaveStyleRule('display', 'none', {
+      media: '(max-width: 768px)',
+    });
+  });
```

Comment:
zustand raises log errors when running this test
```
      Warning: It looks like you're using the wrong act() around your test interactions.
      Be sure to use the matching version of act() corresponding to your renderer:
      
      // for react-dom:
      import {act} from 'react-dom/test-utils';
      // ...
      act(() => ...);
      
      // for react-test-renderer:
      import TestRenderer from react-test-renderer';
      const {act} = TestRenderer;
      // ...
      act(() => ...);
          at /Users/vassili/repositories/marsha/src/frontend/node_modules/grommet/components/Grommet/Grommet.js:69:24
          at LiveVideoPanel (/Users/vassili/repositories/marsha/src/frontend/components/LiveVideoPanel/index.tsx:19:34)
          at IntlProvider (/Users/vassili/repositories/marsha/src/frontend/node_modules/react-intl/src/components/provider.js:81:47)
```

I have not found a proper solution yet

Reply:
It's because we use the `act` from `react-testing-library` in the zustand mock reseting every stores after each tests and here you are using the `react` renderer. 

https://github.com/openfun/marsha/blob/master/src/frontend/__mocks__/zustand.js#L26-L28

Reply:
I apologize, we are not using `act` from `react-testing-library` but from `react-dom/test-utils`. But I think the error is still caused by the different renderer you are using

Reply:
https://github.com/openfun/marsha/pull/1248/commits/bf5d3ac9a7cbeff4a2fe04f09e84fd6070f66b93